src/modules/Resources/Devices.ts

```diff
@@ -59,9 +59,7 @@ class Devices extends TagoIOModule<GenericModuleParams> {
       },
     });
 
-    result = result.map((data) =>
-      dateParser(data, ["last_input", "last_output", "updated_at", "created_at", "inspected_at"])
-    );
+    result = result.map((data) => dateParser(data, ["last_input", "updated_at", "created_at"]));
```

Comment:
`last_output`, `inspected_at` and `last_retention` are not returned by Device info route, so they were removed.